src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaServiceTest.java

```diff
@@ -52,37 +53,48 @@
     private CamundaErrorDecoder camundaErrorDecoder;
 
     @Mock
-    private CamundaQueryBuilder camundaQueryBuilder;
+    private CreateHmctsTaskVariable createHmctsTaskVariable;
 
     @Mock
-    private TaskMapper taskMapper;
+    private CamundaQueryBuilder camundaQueryBuilder;
 
     private CamundaService camundaService;
 
     @BeforeEach
     public void setUp() {
+
         camundaService = new CamundaService(
             camundaServiceApi,
             camundaQueryBuilder,
-            taskMapper,
-            camundaErrorDecoder
+            camundaErrorDecoder,
+            createHmctsTaskVariable
         );
     }
 
     @Test
     void getTask_should_succeed() {
 
         String taskId = UUID.randomUUID().toString();
+        ZonedDateTime dueDate = ZonedDateTime.now().plusDays(1);
+        CamundaTask camundaTask = new CamundaTask(
+            taskId,
+            "someTaskName",
+            "someAssignee",
+            ZonedDateTime.now(),
+            dueDate,
+            null,
+            null,
+            "someFormKey"
+        );
 
-        CamundaTask mockedTask = mock(CamundaTask.class);
-        when(camundaServiceApi.getTask(taskId)).thenReturn(mockedTask);
+        camundaService.getTask(taskId);
 
-        CamundaTask response = camundaService.getTask(taskId);
+        verify(camundaServiceApi, times(1)).getTask(eq(taskId));
+        verify(camundaServiceApi, times(1)).getVariables(camundaTask.getId());
 
-        verify(camundaServiceApi, times(1)).getTask(taskId);
         verifyNoMoreInteractions(camundaServiceApi);
 
-        assertEquals(mockedTask, response);
```

Comment:
We should be mocking the call the camundaServiceApi, then the call to createHmctsTaskVariable.mapToTaskObject and checking the response is what is returned by this method.